pilz_teleoperation/src/pilz_teleoperation/ros_message_serializer.py

```diff
@@ -0,0 +1,53 @@
+import genpy
+import rospy
+
+class RosMessageSerializer:
+    """
+    Serialization and De-Serialization of ROS messages for convenient file printing
+    """
+    def __init__(self):
+        self._module_imports = {}
+
+    def write_messages_to_file(self, ros_messages, filename):
+        """
+        Writes a file that can be used with 'import' to restore
+        :param ros_messages: iterable of messages to be serialized.
+                tuple contains the variable name used for serialization and a ROS message object
```

Comment:
why using tuple and not dict since its supposed to be a key, value pair and dict is exactly that.